core/loader/utils.ts

```diff
@@ -61,6 +61,36 @@ export function findAnchorHrefs(
     .map(a => buildFullURL(a, text.url))
 }
 
+/**
+ * Returns an array of links found in the Link http header of a website,
+ * if the header is present.
+ * An example of a Link header with multiple urls:
+ * <http://blog.com/?feed=rss2>; rel="alternate"; type="application/rss+xml"
+ * Urls can also be multiple, comma-separated. And possibly relative.
+ */
+export function findHeaderLinks(
+  response: TextResponse,
+  type: string
+): string[] {
+  let linkHeader = response.headers.get('Link')
+  if (!linkHeader) {
+    return []
+  }
+  return linkHeader.split(/,\s?/).reduce((urls: string[], link: string) => {
```

Comment:
```suggestion
  return linkHeader.split(/,\s?/).reduce<string[]>((urls, link) => {
```

By using type template we can avoid casting types for callback arguments.

Reply:
Roger that